Stateless/Stateless.project.xproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@ArturKarbone I think this is a leftover file from the conversion, is that the case?